### ISSUE TITLE
Test accuracy with different batch sizes

### ISSUE THREAD

**Issue by stillwaterman** [qubvel/segmentation_models.pytorch#128]
Hi, nice work. I found a problem, when I use unet+seresnet50 to train with my data, I use the model to predict one image, but the seg result is different from I use the model to predict a batch images(contain that image)? I can't find out where the problem is? Could you give me some advice?

**Comment by stillwaterman** [comment]
my code error

**Comment by edwardcho** [comment]
Hello Sir,

I tested sample code for inference..
I had succeed 1 image inference.. But I couldn't succeed batch-inference.
If you don't mind, please tell me hint for batch-inference.....

Thanks...

**Comment by kaelsunkiller** [comment]
@stillwaterman I have encountered the same problem like your description, could you please share your solution? 
My usage of this  library was simple, just like the tutorial, as `smp.Unet()`. No exctra specific parameters, but got the different outputs of the model with different batch size(no other processing, data not shuffled).  The only thing may make me confused is that I trained the model on two gpus, but I evaluate it on a single device. Does this go wrong?

**Comment by qubvel** [comment]
Didn't you forget to call `model.eval()` before inference ?

**Comment by kaelsunkiller** [comment]
OH Yes, I forgot that thing, simple mistake. Sorry for taking your time, Thanks a lot!
